ansible/server-state-playbooks/www-dev/www-dev.yml

```diff
@@ -0,0 +1,162 @@
+      yum:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - ruby
+        - ruby-devel
+        - zlib-devel
+        - "@Development tools"
+
+    - name: Jekyll/Travis Prerequisites | Prerequisite Gems
+      become: yes
+      gem:
+        name: jekyll-feed
+        version: 0.7
+
+    - name: Jekyll/Travis Prerequisites | Prerequisite Gems
+      become: yes
+      gem:
+        name: nokogiri
+        version: 1.6.8.1
+
+    - name: Jekyll/Travis Prerequisites | Prerequisite Gems
+      become: yes
+      gem:
+        name: activesupport
+        version: 4.2.9
+
+    - name: Jekyll/Travis Prerequisites | Prerequisite Gems
+      become: yes
+      gem:
+        name: html-proofer
+        version: 3.2.0
+
+  roles:
+    # Now OME are using RHEL without Spacewalk, the current best-method of
+    # checking `is server deployed in Dundee/SLS` is checking for the SLS nameservers.
+    - role: openmicroscopy.system-monitor-agent
+      tags: monitoring
+      when: "'10.1.255.216' in ansible_dns.nameservers"
+
+    - role: openmicroscopy.nginx
+
+    - role: openmicroscopy.jekyll-build
+      tags: jekyll
+      jekyll_build_git_repo: "https://github.com/openmicroscopy/www.openmicroscopy.org"
+      jekyll_build_force_rebuild: True
+   
+  post_tasks:
+
+    # TODO define this in a variable from host_vars, as varies per-host
+    # and we are also using it in the config for role openmicroscopy.jekyll-build
+    - name: Deployed HTML Symlink
+      become: yes
+      file:
+        src: /var/www/localhost/html/
+        path: /var/www/localhost/html/www.openmicroscopy.org
+        state: link
+
+    - name: NGINX - Performance tuning - worker processes
+      become: yes
+      replace:
+        path: "/etc/nginx/nginx.conf"
+        regexp: '^worker_processes\s+\d+;'
+        replace: "worker_processes {{ ((ansible_processor_count * ansible_processor_cores) / 2) |round|int }};"
+
+    # cf https://www.digitalocean.com/community/tutorials/how-to-optimize-nginx-configuration
+    - name: NGINX - Performance tuning - worker connections
+      become: yes
+      replace:
+        path: "/etc/nginx/nginx.conf"
+        regexp: 'worker_connections\s+\d+;'
+        replace: "worker_connections 65000;"
+
+    - name: NGINX - SSL File Deployment - prepare directory
+      become: yes
+      file:
+        path: "{{ nginx_ssl_files_path }}"
+        state: directory
+        owner: root
+        group: root
+        mode: "u=r,go=" 
+
+    - name: NGINX - SSL File Deployment
+      become: yes
+      copy:
+        dest="{{ item.key }}"
+        content="{{ item.value.content }}"
+        owner="{{ item.value.owner }}"
+        group="{{ item.value.group }}"
+        mode="{{ item.value.mode }}"
+      with_dict: "{{ nginx_ssl_cert_files }}"
+      no_log: true
+
+    - name: NGINX config | www-dev.openmicroscopy.org
+      become: yes
+      template:
+        src: templates/www-dev.openmicroscopy.org.conf.j2
```

Comment:
Yes, I had intended to add the config here via the template too. Missed in amongst the other (intentionally) not-included files. Thanks for noticing.